### PR TITLE
Create and update cs.po

### PR DESCRIPTION
**Pull request by MultiSeptim**
My first step to the czech translation of this campaign (as you wished some time ago). I have a little more time now, so I have decided to take this task. I am not sure how quickly I will able to work on it, I prefer to translate more aNO for example, but be sure I will work on it. Slowly but surely...

@celerini it would be nice if you could check it, please.

### CODE REVIEW

**Comment by MultiSeptim** [comment]
And some units from this campaign (e.g. some Woses) are included in mainline now...

**Comment by cooljeanius** [comment]
> And some units from this campaign (e.g. some Woses) are included in mainline now...

oh yeah I think @Konrad22 was pointing that out, too...

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 1]
```diff
@@ -0,0 +1,9841 @@
...
+
+#. [dummy]: id=teg_ability_cave_experience
+#: The_Earths_Gut/utils/abilities.cfg:46
+msgid ""
+"This unit has some experience traveling through dungeons: Movement costs are "
+"decreased by 1 and defense increased by 10% for terrain types cave and "
+"fungus."
+msgstr ""
+"Tato jednotka nabrala určité zkušenosti s cestováním skrz podzemí. V "
+"jeskynních a houbových terénem ji pohyb stojí o 1 pohybový bod méně a její "
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:360]:
```suggestion
"jeskynních a houbových terénech ji pohyb stojí o 1 pohybový bod méně a její "
```

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 2]
```diff
@@ -0,0 +1,9841 @@
...
+"a Master in battle is considered an honor reserved for only the bravest of "
+"dwarvish warriors."
+msgstr ""
+
+#. [attack]: type=impact
+#: The_Earths_Gut/units/dwarves/Master_of_Alchemy.cfg:45
+#: The_Earths_Gut/units/dwarves/Alchemist.cfg:42
+#: The_Earths_Gut/units/dwarves/Potion_Smith.cfg:42
+msgid "morningstar"
+msgstr "řemdich"
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:872]:
```suggestion
msgstr "řemdih"
```

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 3]
```diff
@@ -0,0 +1,9841 @@
...
+
+#. [unit_type]: id=Pirate Ship, race=mechanical
+#: The_Earths_Gut/units/ships/Pirate_Ship.cfg:24
+msgid ""
+"Pirate Ships are the wolves of the sea, and transport ships are their sheep. "
+"The pirates who crew these ships are more than happy to relieve other ships "
+"of their cargo."
+msgstr ""
+"Pirátské lodě jsou jako vlci moře a transportní lodě jako jejich ovce. "
+"Piráti, kteří tvoří posádu těchto lodí, jsou více než rádi, když mohou "
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:1075]:
```suggestion
"Piráti, kteří tvoří posádku těchto lodí, jsou více než rádi, když mohou "
```

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 4]
```diff
@@ -0,0 +1,9841 @@
...
+#: The_Earths_Gut/scenarios/15_Save_the_heir.cfg:114
+#: The_Earths_Gut/scenarios/15_Save_the_heir.cfg:151
+#: The_Earths_Gut/scenarios/12_Under_pressure.cfg:1173
+msgid "Hurry up!"
+msgstr "Pospěšme!"
+
+#. [message]: id=Elara
+#: The_Earths_Gut/scenarios/15_Save_the_heir.cfg:122
+msgid "Quick, Hamel!"
+msgstr "Rychle, Hameli!"
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:6627]:
```suggestion
msgstr "Rychle, Hamele!"
```

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 5]
```diff
@@ -0,0 +1,9841 @@
...
+
+#. [message]: id=Hamel
+#: The_Earths_Gut/scenarios/05b_Unknown_tunnels.cfg:1043
+msgid "Too bad. It might have been the only way out..."
+msgstr "To je moc špatné. Možná to byla jediná cesta ven..."
+
+#. [message]: id=Clarissa
+#: The_Earths_Gut/scenarios/05b_Unknown_tunnels.cfg:1047
+msgid "There must be another way, either to the north or south."
+msgstr "Musí být nějaká jiná cesta, buď na sever nebo na jih."
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:7360]:
```suggestion
msgstr "Musí být nějaká jiná cesta, buď na sever, nebo na jih."
```

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 6]
```diff
@@ -0,0 +1,9841 @@
...
+#: The_Earths_Gut/scenarios/01t_The_old_home.cfg:42
+msgid ""
+"But it was not to last, as bands of greedy orcs lusted over and vowed to "
+"take the riches of Dwarven Doors for themselves. Blinded by their comforts "
+"and security, the inhabitants of the city were unprepared for the raids and "
+"slaughters that followed. Nearly as fast as Dwarven Doors had risen to its "
+"prime, the proud city fell to the savage hordes."
+msgstr ""
+"Ale nemělo to dlouhé trvání, neboť tlupy chamtivých skřetů zatoužily po "
+"bohatství Trpasličích Dveří a přísahali, že ho získají pro sebe. Obyvatelé "
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:7496]:
```suggestion
"bohatství Trpasličích Dveří a přísahaly, že ho získají pro sebe. Obyvatelé "
```

File: translations/wesnoth-The_Earths_Gut/cs.po [hunk 7]
```diff
@@ -0,0 +1,9841 @@
...
+msgid ""
+"Forced to retreat ever deeper into the hostile mountains, the dwarves banded "
+"together under their new leader Lord Relgorn. They fought battle after "
+"bloody battle with the greedy orcs, and eventually escaped into the deeper "
+"caverns where few dared to follow."
+msgstr ""
+"Trpaslíci, nuceni ustupovat stále hlouběji do nepřátelských hor, se "
+"sjednotili pod svým novým vůdcem lordem Relgornem. Vybojovali se skřety "
+"jednu krvavou bitvou za druhou a nakonec unikli do hlubších jeskyní, kam se "
+"jen málokdo odvážil následovat."
```
Comment (celerini) [inline translations/wesnoth-The_Earths_Gut/cs.po:7523]:
```suggestion
"jen málokdo odvážil je následovat."
```

**Review by celerini — CHANGES_REQUESTED** [review]
great work, @MultiSeptim

**Comment by MultiSeptim** [comment]
Ready to merge.

**Pull request merged**